server/src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -64,7 +66,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String userEmail = userService.getCurrentUser().getEmail();
     String aboutMe = request.getParameter("text");
 
-    User user = new User(userEmail, aboutMe);
+
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+
```

Comment:
nit-picky: seems like there are a lot of extra lines here. We should compress the code to at most 1 extra line between lines.